quarkus-opentelemetry-exporter-azure/deployment/src/main/java/io/quarkiverse/opentelemetry/exporter/azure/deployment/AzureExporterProcessor.java

```diff
@@ -64,9 +64,13 @@ void runtimeInitializedClasses(BuildProducer<RuntimeInitializedClassBuildItem> r
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem("io.netty.internal.tcnative.SSL"));
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem("io.netty.util.concurrent.GlobalEventExecutor"));
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem(
-                "com.azure.core.http.vertx.VertxAsyncHttpClientProvider$GlobalVertxHttpClient"));
+                "com.azure.core.http.vertx.VertxHttpClientProvider$GlobalVertxHttpClient"));
```

Comment:
@ninckblokje Is it related to https://github.com/quarkiverse/quarkus-opentelemetry-exporter/pull/328#discussion_r2073684561?

By replacing `VertxAsyncHttpClientProvider` with `VertxHttpClientProvider`, I am a bit afraid that it could work in your project but not in other ones.

Reply:
Yeah, this class was renamed: https://github.com/Azure/azure-sdk-for-java/commit/0b65b4d4771f00e1f4260897962cd3e5d487894e

Reply:
Classes have been renamed on Azure side: https://github.com/Azure/azure-sdk-for-java/pull/42072, explaining the need to change the GraalVM configurations.